packages/lwc-language-server/src/javascript/__tests__/compiler.test.ts

```diff
@@ -15,86 +14,18 @@ import {
     getPrivateReactiveProperties,
     getProperties,
     getPublicReactiveProperties,
-    extractLocationFromBabelError,
-    extractMessageFromBabelError,
 } from '../compiler';
 
 const codeOk = `
 import { LightningElement } from 'lwc';
 export default class Foo extends LightningElement {}
 `;
-
-function pretify(str: string): string {
-    return str
-        .toString()
-        .replace(/^\s+|\s+$/, '')
-        .split('\n')
-        .map(line => line.trim())
-        .filter(line => line.length)
-        .join('\n');
-}
-
-it('can use transform from lwc-compiler', async () => {
-    const expected = `
-        import _tmpl from "./foo.html";
-        import { registerComponent as _registerComponent } from "lwc";
-        import { LightningElement } from 'lwc';
-        class Foo extends LightningElement {}
-        export default _registerComponent(Foo, {
-        tmpl: _tmpl
-        });
-    `;
-
-    const options: CompilerOptions = {
-        name: 'foo',
-        namespace: 'x',
-        files: {},
-    };
-    const transformerResult = await transform(codeOk, 'foo.js', options);
-    const code = transformerResult.code;
-    expect(pretify(code)).toBe(pretify(expected));
-});
```

Comment:
A number of these tests made assertions directly against the ancient LWC compiler. That's not particularly useful. It also cannot be left in place, since metadata is no longer exposed via the LWC compiler but through `@lwc/metadata` instead.